data_collection/timeprof_matrix_bot.py

```diff
@@ -236,8 +239,8 @@ def __init__(self, homeserver, mid, bot_pw):
             homeserver,
             mid,
             ssl=True,
-            device_id="matrix-niotest1235",
-            store_path="./store",
+            device_id="matrix-niotest1337",
```

Comment:
To be honest I don't understand how this works. If the STORE_PATH is deleted, a new device_id seems to be needed. Could be investigated further

Reply:
Seems strange to have this hard coded? Is it stored in the STORE_PATH?